cmd/process-agent/collector_api_test.go

```diff
@@ -528,7 +528,7 @@ func runCollectorTest(t *testing.T, check checks.Check, cfg *config.AgentConfig,
 	runCollectorTestWithAPIKeys(t, check, cfg, epConfig, []string{"apiKey"}, []string{"orchestratorApiKey"}, mockConfig, init, tc)
 }
 
-func runCollectorTestWithAPIKeys(t *testing.T, check checks.Check, cfg *config.AgentConfig, epConfig *endpointConfig, apiKeys, orchAPIKeys []string, mockConfig ddconfig.Config, init func(c *Collector), tc func(c *Collector, ep *mockEndpoint)) {
+func runCollectorTestWithAPIKeys(t *testing.T, check checks.Check, _ *config.AgentConfig, epConfig *endpointConfig, apiKeys, orchAPIKeys []string, mockConfig ddconfig.Config, init func(c *Collector), tc func(c *Collector, ep *mockEndpoint)) {
```

Comment:
Since `cfg` is now unused, can we remove it from the function parameters?

Reply:
Ah, never mind it's removed here https://github.com/DataDog/datadog-agent/pull/14889/files#diff-cb0e545e5c9aba9e9985490d4092d686b4444ee971115d6f9ab2624fcf3968baR525